apps/builder/tsconfig.json

```diff
@@ -8,10 +7,14 @@
     "../../@types/**/*.d.ts"
   ],
   "compilerOptions": {
-    "paths": { "~/*": ["app/*"] },
+    "paths": {
+      "~/*": ["app/*"]
+    },
     "baseUrl": ".",
     "rootDir": "./app",
     "declaration": false,
-    "declarationDir": null
+    "emitDeclarationOnly": false,
+    "declarationDir": null,
+    "noEmit": true
```

Comment:
Did remix broke it again?

Reply:
Have no idea, any other values of noEmit cause vscode to show errors on tsconfig

Reply:
Checked with verdaccio on saas, everything works